snapcraft/yaml_utils.py

```diff
@@ -34,7 +34,9 @@ def load(stream: TextIO) -> Any:
 
 def dump(data: Union[Dict[str, Any], yaml.YAMLObject], *, stream: TextIO = None) -> str:
     """Safely dump YAML in ordered manner."""
-    return yaml.dump(data, stream, _SafeOrderedDumper, default_flow_style=False)
+    return yaml.dump(
+        data, stream, _SafeOrderedDumper, default_flow_style=False, allow_unicode=True
```

Comment:
From [the spec](http://yaml.org/spec/1.2/spec.html),
> YAML™ (rhymes with “camel”) is a human-friendly, cross language, Unicode based data serialization language [...]

why on _Earth_ would unicode be disallowed?!? :man_facepalming:

Reply:
yeah, my surprise too, and I only found out about this by reading the source...

Reply:
in their defense, it is allowed, just not represented when dumped :man_shrugging: